.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,8 @@
+blank_issues_enabled: false
+contact_links:
+  - name: PyMC Discourse
+    url: https://discourse.pymc.io/
+    about: Ask usage questions about PyMC/PyMC3
+  - name: Example notebook error report
+    url: https://github.com/pymc-devs/pymc-examples/issues
+    about: Please report errors or desired extensions to the tutorials and examples here.
```

Comment:
```suggestion
```

Do we need this? We can build example notebooks in the docs build and fail it automatically on error: https://github.com/pymc-devs/pymc/pull/6324